task/rhacs-deployment-check/3.71/rhacs-deployment-check.yaml

```diff
@@ -0,0 +1,83 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rhacs-deployment-check
+  labels:
+    app.kubernetes.io/version: "3.71"
+  annotations:
+    tekton.dev/tags: security
+    tekton.dev/categories: Security
+    tekton.dev/displayName: "Policy check a deployment with Red Hat Advanced Cluster Security"
+    tekton.dev/platforms: "linux/amd64"
+    tekton.dev/pipelines.minVersion: "0.18.0"
+spec:
+  description: >-
+    Policy check a deployment with Red Hat Advanced Cluster Security.
+
+    This tasks allows you to check a Kubernetes deployment manifest against policies
+    and apply enforcement to fail pipelines.
+  params:
+    - name: rox_central_endpoint
+      type: string
+      description: Name of secret containing the address:port tuple for RHACS Stackrox Central.
+      default: rox-central-endpoint
+    - name: rox_api_token
+      type: string
+      description: Name of secret containing the RHACS StackRox API token with CI permissions.
+      default: rox-api-token
+    - name: deployment
+      type: string
+      description: |
+        Deployment filename to check.
+        Examples: 'deployment.yaml', '$(workspaces.source.path)/base/deployment.yaml'
+      default: '$(workspaces.source.path)/deployment.yaml'
```

Comment:
can this enable user to provide a `deployment.yaml` which is present at a place which can't be cloned? If so, can we assume that workspace can be made optional in that case?

Reply:
the working dir is set to the workspace path so a relative filename may be used.
i can not think of where the file may come from if not from a git clone in a workspace.

Reply:
> i can not think of where the file may come from if not from a git clone in a workspace.

one such case can be - a file is present in gcs or s3 or any hosted bucket. Hope

Reply:
That does make sense but the tool (roxctl entrypoint) can only operate on POSIX file paths, which puts object stores out of reach of the task independently.

Reply:
I think this is resolved.